src/main/java/cz/startnet/utils/pgdiff/parsers/CreateTriggerParser.java

```diff
@@ -75,6 +75,16 @@ public static void parse(final PgDatabase database,
         final String relationName = parser.parseIdentifier();
 
         trigger.setRelationName(ParserUtils.getObjectName(relationName));
+        
+        String referencing="REFERENCING";
+        if (parser.expectOptional(referencing)) {            
+             trigger.setReferencing("\t"+referencing);             
+
+            while( !parser.getSubString(parser.getPosition()-5, parser.getPosition()-4).equals(System.getProperty("line.separator"))){ 
```

Comment:
this condition is not quite ok, when running the tests on a windows box this fails, I suspect because of "line.separator"
I looked at how the `parseReferencing is constructed and removed the `while` and replaced it with 2 consecutive calls to parseReferencing

Reply:
the relevant section looks like
```
String referencing="REFERENCING";
        if (parser.expectOptional(referencing)) {            
             trigger.setReferencing("\t"+referencing);             
             parseReferencing(parser,trigger);
             parseReferencing(parser,trigger);
        }
```